shell/edit/resources.cattle.io.backup.vue

```diff
@@ -88,7 +88,7 @@ export default {
       this.value['spec'] = { retentionCount: 10 };
     }
     let s3 = {};
-    let useEncryption = false;
+    let useEncryption = true;
```

Comment:
```suggestion
    let useEncryption = this.mode === _CREATE;
```
(and update `import { NAMESPACE, _VIEW, _CREATE } from '@shell/config/query-params';`)

This ensures that editing an existing backup results in the same experience 
1. Users have set encryption, no change (useEncryption overrides this initialisation with true on line 97)
2. Users have not set encryption, no change (not forced to change back to no encryption on every edit)

Reply:
Yeah - this makes sense to me. I've made a commit to correct for this feedback.